notebooks/evaluation/rag_baseline_evaluation.ipynb

```diff
@@ -0,0 +1,3387 @@
+         "execution_count": 32,
+         "metadata": {},
+         "outputs": [],
+         "source": [
+            "from jose import jwt\n",
+            "from uuid import UUID\n",
+            "import requests\n",
+            "import json\n",
+            "import pandas as pd\n",
+            "import pickle\n",
+            "from dataclasses import asdict"
+         ]
+      },
+      {
+         "cell_type": "code",
+         "execution_count": 6,
+         "metadata": {},
+         "outputs": [],
+         "source": [
+            "from dotenv import find_dotenv, load_dotenv\n",
+            "_ = load_dotenv(find_dotenv())"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "[Back to top](#title)"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "-----------------"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "## Start Redbox Running locally <a id=\"run-redbox\"></a>"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "Start docker runtime, either Docker Desktop. However, if you are using colima run the following terminal command\n",
+            "```bash\n",
+            "colima start --memory 8\n",
+            "``` "
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "---------------------"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "#### First-time setup"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "First time users need to do the following"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "```bash\n",
+            "poetry install\n",
+            "```"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "Ensure you .env file has OpenAI API key in and has the following settings:"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "```bash\n",
+            "# === Object Storage ===\n",
+            "\n",
+            "MINIO_HOST=minio\n",
+            "MINIO_PORT=9000\n",
+            "MINIO_ACCESS_KEY=minioadmin\n",
+            "MINIO_SECRET_KEY=minioadmin\n",
+            "AWS_ACCESS_KEY=minioadmin\n",
+            "AWS_SECRET_KEY=minioadmin\n",
+            "\n",
+            "AWS_REGION=eu-west-2\n",
+            "\n",
+            "# minio or s3\n",
+            "OBJECT_STORE=minio\n",
+            "BUCKET_NAME=redbox-storage-dev\n",
+            "```"
```

Comment:
Should we say something about making sure `EMBEDDING_MODEL` is set to something reasonable (and consistent?)